docs/deployment/migration-guide.md

```diff
@@ -23,6 +23,7 @@
   Both Derby and SQLite are mainly for testing purposes, and they're not supposed to be used in production.
   To restore previous behavior, set `kyuubi.metadata.store.jdbc.database.type=DERBY` and
   `kyuubi.metadata.store.jdbc.url=jdbc:derby:memory:kyuubi_state_store_db;create=true`.
+* Since Kyuubi 1.8, if the directory of the embedded zookeeper configuration is a relative path, it is resolved relative to `$KYUUBI_HOME` instead of `$PWD`.
```

Comment:
could we mention the exact configuration key in the sentence to avoid confusing users?

Reply:
thx, added